website/content/docs/connect/cluster-peering/usage/establish-cluster-peering.mdx

```diff
@@ -173,14 +170,48 @@ You must use the Consul CLI to complete this step. The HTTP API and the Consul U
 
   </CodeBlockConfig>
 
-1. Add the configuration entry to your cluster.
+1. Add the configuration entry to your cluster. If necessary, include the `-partition` flag to write the configuration entry to a non-default admin partition.
 
   ```shell-session
   $ consul config write peering-config.hcl
   ```
 
 Before you proceed, wait for the clusters to sync and make services available to their peers. To check the peered cluster status, [read the cluster peering connection](/consul/docs/connect/cluster-peering/usage/manage-connections#read-a-peering-connection).
 
+</Tab>
+<Tab heading="HTTP API" group="api">
+
+1. Create a configuration entry and specify the `Kind` as `"exported-services"`. The following example exports services between `default` admin partitions. To export services between non-default admin partitions, you must include a `"Partition"` filed that contains the name of the local partition the configuration entry applies to.
```

Comment:
Should we explicitly include the `Partition` parameter in here and all related sections? I believe it's better safe than sorry but i'm not sure if Consul CE will complain about the parameter.